knapsack-gosdk/go.mod

```diff
@@ -1,8 +1,14 @@
 module example.com/your_project/knapsack-gosdk
 
-go 1.19
+go 1.21
```

Comment:
Shouldn’t this be already `go 1.22`?

Reply:
We only need go 1.21 as far as I am aware, and we don't want to exclude users of 1.21.

Reply:
Yes, for the "engines" we should use the oldest, still supported, major version. However, this is an application that _uses_ the engine. Shouldn’t we be in the "forefront" here?

Reply:
Fair point, let me bump it. :relaxed: